### ISSUE TITLE
no-input mode or "assume yes mode" required for some pip commands

### ISSUE THREAD

**Issue by NorthIsUp** [Yelp/pip-faster#45]
this showed up in my ci logs:
```
Obtaining gutter from git+https://github.com/disqus/gutter.git@eb8285ea2fa072d11699fac6c55a202591e14b6d#egg=gutter (from -r requirements/global.txt (line 9))
  git clone in ./env/src/gutter exists with URL https://github.com/dpetzold/gutter.git
  The plan is to install the git repository https://github.com/disqus/gutter.git
What to do?  (s)witch, (i)gnore, (w)ipe, (b)ackup Cleaning up…
```

This command needs to be run with some way to force wipe or something for headless servers.

**Comment by NorthIsUp** [comment]
fixed by #46

**Comment by bukzor** [comment]
Looking more closely, can you explain how you got into this situation? It looks like you cloned the same package from two different places. I wouldn't expect this to happen on a single run, and venv-update's contract is to put you in a state as if you rm-rf'd and re-built from scratch

**Comment by NorthIsUp** [comment]
It happened over two runs.

We had an outstanding pull request on a library but that code was needed for us to keep moving on. so we had the following in the requirements.txt

```
-e git+htts://github.com/myfork.git@mysha#egg=myegg
```

after the feature was merged in we changed it to

```
-e git+htts://github.com/mainline.git@mysha#egg=myegg
```

this is when the error occurred.

In testing the issue only occurs when switching repo urls, it does not seem to be an issue when going from a repo to `mainline==1.0.0`


-- 
Adam Hitchcock
Sent with Airmail

On February 6, 2015 at 4:08:05 PM, Buck Evan (notifications@github.com) wrote:

Looking more closely, can you explain how you got into this situation? It looks like you cloned the same package from two different places. I wouldn't expect this to happen on a single run, and venv-update's contract is to put you in a state as if you rm-rf'd and re-built from scratch

**Comment by bukzor** [comment]
I believe you said that -e urls don't work. Did you patch it locally, or it
works just sometimes?

**Comment by NorthIsUp** [comment]
I got to the bottom of #36.

**Comment by chriskuehl** [comment]
@NorthIsUp This should be fixed in 1.0rc2. 

We've separated pip-faster from venv-update, which has the same interface as pip.
It simply runs in a subprocess of venv-update, so the prompt should work per usual in this case.

Double check for us?
